### PR TITLE
[TRAFODION-2659]update win ODBC/ODB version number to 2.2.0

### PR DESCRIPTION
**Pull request by xwq**

### CODE REVIEW

**Comment by hegdean** [comment]
Changes look ok

File: win-odbc64/odbcclient/drvr35msg/DrvMsg35.rc [hunk 1]
```diff
@@ -71,7 +71,7 @@ BEGIN
             VALUE "FileDescription", "TRAF ODBC Client Msg DLL"
             VALUE "FileVersion", "1.3.0.0"
```
Comment (arvind-narain) [inline win-odbc64/odbcclient/drvr35msg/DrvMsg35.rc]: FileVersion and ProductVersion to be updated to 2.2 in this file too ?

**Comment by xwq** [comment]
@arvind-narain the file missed in the script , I update it and add it to the script.

**Comment by arvind-narain** [comment]
Thanks +1

**Pull request merged**